subtract_numbers.py

```diff
@@ -0,0 +1,4 @@
+def subtract_numbers(a, b):
+    return a - b
+
+print(subtract_numbers(10, 5))
\ No newline at end of file
```

Comment:
Avoid calling functions and printing results directly in the global scope of your script. This makes the code less modular and harder to test. Consider using a main function to call your other functions.